frontend/src/reducers/addLead.js

```diff
@@ -63,13 +63,16 @@ export default function(state = initialState, action) {
       }
 
     case types.ADD_LEAD_HANDLE_FORM_CHANGE:
+      delete state.errors[action.payload.name]
```

Comment:
Please avoid mutating the previous state

Reply:
Thanks, it's corrected now,